File: Java/1929.java
class Solution{
    public int[] getConcatenation(int[] nums){

        // Solution 1
        // Using Modulo
        
        int n = nums.length;
        int[] ans = new int[n*2];

        // base case
        if ( n == 0){
            return ans;
        }

        for (int i = 0; i < n*2; i++){
            ans[i] = nums[i%n];
        }

        return ans
    }
}

File: Python/1929.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:

        # Solution 1
        # 

        n = len(nums)
        ans = []

        # base case
        if n == 0:
            return ans

        for i in range(n*2):
            ans.append(nums[i%n])

        return ans
